### ISSUE TITLE
Re-write the Kinect information tree view.

### ISSUE THREAD

**Issue by cantsin** [SixSecondMonks/sos-display#44]
The tree view as it stands is suboptimal; I created a generic tree structure to mirror any arbitrary JS dictionary but I realized that all we need is a table of depth 2 at most. Implementing this should also lead to a performance improvement.

**Comment by timkettering** [comment]
agreed.  i also hacked on it a bit to display the skeleton color in background but it makes a messy dependency on `Overlay`, so this prob needs to go hand-in-hand with creating a specific store for Kinect data, that also contains info about the skeleton on the overlay.
